auth0/src/main/java/com/auth0/android/authentication/storage/CredentialsManager.java

```diff
@@ -140,6 +164,37 @@ public void onFailure(@NonNull AuthenticationException error) {
                 callback.onFailure(new CredentialsManagerException("An error occurred while trying to use the Refresh Token to renew the Credentials.", error));
             }
         });
+
+    }
+
+    private boolean hasScopeChanged(@NonNull String storedScope, @Nullable String requiredScope) {
+        if (requiredScope == null) {
+            return false;
+        }
+        String[] stored = storedScope.split(" ");
+        Arrays.sort(stored);
+        String[] required = requiredScope.split(" ");
+        Arrays.sort(required);
+        return stored != required;
```

Comment:
Shouldn't this be `!Arrays.equals(stored, required)`? With `!=` the references will be compared, not the contents, AFAIK.

Reply:
In Auth0.swift these methods are tested individually: https://github.com/auth0/Auth0.swift/blob/master/Auth0Tests/CredentialsManagerSpec.swift#L300, https://github.com/auth0/Auth0.swift/blob/master/Auth0Tests/CredentialsManagerSpec.swift#L324

Reply:
Good catch. Yes, I should be using that util method.